src/main/java/com/homeworks/homework20/service/SchedulerService.java

```diff
@@ -0,0 +1,41 @@
+package com.homeworks.homework20.service;
+
+import com.homeworks.homework20.model.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SchedulerService {
+    private static final String CRON = "*/10 * * * * *";
+
+    private final UserService userService;
+    private final EmailService emailService;
+
+    @Scheduled(cron = CRON)
+    public void sendMailtoUsers() {
+        LocalDate date = LocalDate.now();
+        int month = date.getMonthValue();
+        int day = date.getDayOfMonth();
+        List<User> usersBirthdays = userService.findAllUsersByBirthDay(month, day);
+        if (!usersBirthdays.isEmpty()) {
```

Comment:
Do we need this check or not?

Reply:
?

Reply:
I'm not sure what will happen if the List will be empty - removed this check in the new commit